src/NUnitFramework/framework/Attributes/TestCaseAttribute.cs

```diff
@@ -387,6 +414,20 @@ public IEnumerable<TestMethod> BuildFrom(IMethodInfo method, Test suite)
         {
             TestMethod test = new NUnitTestCaseBuilder().BuildTestMethod(method, suite, GetParametersForTestCase(method));
 
+            if (_untilDate.HasValue)
+            {
+                if (_untilDate > DateTimeOffset.UtcNow)
+                {
+                    test.RunState = RunState.Ignored;
+                    string reason = string.Format("Ignoring until {0}. {1}", _untilDate.Value.ToString("u"), IgnoreReason);
```

Comment:
Please use the C# 6 interpolated string feature:

```suggestion
                    string reason = $"Ignoring until {_untilDate.Value:u}. {IgnoreReason}";
```

Is there anywhere else that we're building an "Ignoring until" message? If so, we should deduplicate as much as possible into one of our formatting helpers.

Reply:
“Please use the C# 6 interpolated string feature:”

Why?

This is not part of our code no standards, as far as I know.

Many coding shops have rejected the use of the interpolated string feature.

There is no reason to impose it here.

Reply:
@oznetmaster I don't agree with your assumptions and the options you're effectively leaving seem frustrating to me. Out of courtesy to Andrew, this is not a good place to have a long conversation. Would you be willing to discuss at https://github.com/orgs/nunit/teams/framework-team or over email? It would be appreciated.

@abborg No worries about the interpolated string :D

Reply:
I would agree that this is not the place for discussions like this. I'd like to keep PRs on-point and not distract from the code being discussed. Deeper coding philosophical issues can be discussed in other channels.